### ISSUE TITLE
Add my Dataverse repository: RSU Dataverse

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse-installations#98]
**Name of repository:**
RSU Dataverse

**Homepage URL of repository:**
https://dataverse.rsu.lv/

**Location of the repository (city or street address if possible):**
Dzirciema iela 16, Riga, Latvia (56.95435220722525, 24.055010284655705)

*The following isn't required but is nice to have:*

**A short description (few sentences) about the repository:**
The Riga Stradins University Research Data Repository preserves, disseminates, and provides access to research data generated by its RSU scholars and researchers, in order to give visibility to such data, ensure its preservation and facilitate its access and reuse and comply with FAIR principles.

**Email address of repository contact:**

**Year when repository started operating:**
2020

**DOI authority (if applicable):**

**Link to successful [Core Trust Seal](https://www.coretrustseal.org) application:**

**Comment by jggautier** [comment]
I'm waiting for confirmation on the location of the repository.

**Comment by jggautier** [comment]
Location and description added in this issue and in the spreadsheet.

@pdurbin, I get these errors when running the update-data.py:

<details>
<summary>Error messages:</summary>

```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1317, in do_open
    encode_chunked=req.has_header('Transfer-encoding'))
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1016, in _send_output
    self.send(msg)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 956, in send
    self.connect()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1392, in connect
    server_hostname=server_hostname)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/ssl.py", line 412, in wrap_socket
    session=session
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/ssl.py", line 850, in _create
    self.do_handshake()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/ssl.py", line 1108, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:1045)

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "update-data.py", line 14, in <module>
    response = urlrequest.urlopen(metrics_url)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 525, in open
    response = self._open(req, data)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 543, in _open
    '_open', req)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1360, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1319, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:1045)>
``` 

</details>

I remember this happening before and think it was resolved by contacting the installation. Is this an issue with how their installation is configured? If so, could I forward the support email to you so that you could let the installation admin know what to do?

Or maybe it's an issue with my version of python or OpenSSL library (that's what I think is being suggested in https://stackoverflow.com/questions/44316292/ssl-sslerror-tlsv1-alert-protocol-version)?

**Comment by shlake** [comment]
@jggautier it works for me.

**Comment by jggautier** [comment]
Hmmm okay! I'm installing updates on my computer now and I'll try running the script again today when updates are done. If the updates don't let me run the script, I'll let you know so you can commit your change. Thanks for checking!

**Comment by jggautier** [comment]
Updating my MacOS didn't work. Not sure what else to do.

@shlake could you commit your change, then?

> There are two hosts the script tries to contact (and it would be good to figure out if one or both are failing):
> 
> Google Sheets
> The metrics server run by UNC

I think the error occurs when the script tries to open the metrics_url (https://dataversemetrics.odum.unc.edu/dataverse-metrics/config.json)

I can open that url  in my browser, but when I try to run...

```
import urllib.request as urlrequest

metrics_url = 'https://dataversemetrics.odum.unc.edu/dataverse-metrics/config.json'
response = urlrequest.urlopen(metrics_url)
```
... I get the same error message I posted earlier.

**Comment by shlake** [comment]
Closed by #99

**Comment by pdurbin** [comment]
@shlake looks great! Thank you!

<img width="693" alt="Screen Shot 2021-04-27 at 11 46 33 AM" src="https://user-images.githubusercontent.com/21006/116271897-6c3a9b80-a74e-11eb-94bb-fc7bd8fa5690.png">

@jggautier I'll ping you about your Python woes.
